pipeline.yaml

```diff
@@ -11,38 +11,13 @@
 # files are properly formatted.
 
 template: "regression/v1"
-# Specifies the dataset to use for model development
-data:
-  # Dataset locations on the local filesystem are supported, as well as HTTP(S) URLs and
-  # any other remote locations resolvable by MLflow, such as those listed in
-  # https://mlflow.org/docs/latest/tracking.html#artifact-stores
-  location: {{INGEST_DATA_LOCATION}}
-  # Beyond `parquet` datasets, the `spark_sql` and `delta` formats are also natively supported for
-  # use with Spark
-  format: {{INGEST_DATA_FORMAT|default('parquet')}}
-  # Datasets with other formats, including `csv`, can be used by implementing and
-  # specifying a `custom_loader_method`
-  custom_loader_method: steps.ingest.load_file_as_dataframe
-  # If the `spark_sql` `format` is specified,
-  # And if the table location format is path-like, use the following sql command for Spark to read
-  # sub-columns from the table:
-  sql: SELECT * FROM delta.`{{INGEST_DATA_LOCATION}}`
-  # And if the table location format is table-like, use the following sql command for Spark to read
-  # sub-columns from the table:
-  # sql: SELECT col1, col2 FROM {{INGEST_DATA_LOCATION}}
-  # If the `delta` `format` is specified, you can also configure the Delta table `version` to read
-  # or the `timestamp` at which to read data
-  # version: 2
-  # timestamp: 2022-06-01T00:00:00.000Z
-# specify the dataset to use for batch scoring.  All params serve the same function as in `data`
-data_scoring:
-  location: {{INGEST_SCORING_DATA_LOCATION}}
-  format: {{INGEST_SCORING_DATA_FORMAT|default('parquet')}}
-  custom_loader_method: steps.ingest.load_file_as_dataframe
-  sql: SELECT * FROM delta.`{{INGEST_SCORING_DATA_LOCATION}}`
 # Specifies the name of the column containing targets / labels for model training and evaluation
 target_col: "fare_amount"
+# Sets the primary metric to use to evaluate model performance. This primary metric is used
+# to sort MLflow Runs corresponding to the pipeline in the MLflow Tracking UI
+primary_metric: "root_mean_squared_error"
 steps:
+  ingest: {{INGEST_STEP_CONFIG}}
```

Comment:
Could simplify the variable name a bit: `INGEST_STEP_CONFIG`-> `INGEST_CONFIG`
ditto below for other step configs.

Reply:
yup we can. 👍🏻